perfkitbenchmarker/benchmarks/aerospike_ycsb_benchmark.py

```diff
@@ -128,8 +128,10 @@ def Run(benchmark_spec):
 
   metadata = {'ycsb_client_vms': FLAGS.ycsb_client_vms,
               'num_vms': len(aerospike_vms),
-              'scratch_disk_type': FLAGS.scratch_disk_type,
-              'scratch_disk_size': FLAGS.scratch_disk_size}
+              'scratch_disk_type': FLAGS.disk_type,
```

Comment:
Since the disk type and size can be specified by configs, FLAGS.disk_type may not actually be the disk type used by the benchmark (same for size etc).